include/GoogleSync/GoogleSync.php

```diff
@@ -834,10 +1002,13 @@ public function updateSuitecrmMeetingEvent($event_local, $event_remote)
      *
      * @return Meeting SuiteCRM Meeting Bean
      */
-    public function createSuitecrmMeetingEvent($event_remote)
+    public function createSuitecrmMeetingEvent(Google_Service_Calendar_Event $event_remote)
     {
-        $GLOBALS['log']->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Creating New SuiteCRM Meeting');
-        $meeting = new Meeting;
+        $this->logger->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Creating New SuiteCRM Meeting');
+        $meeting = BeanFactory::getBean('Meetings');
+        if (!$meeting) {
+            throw new Exception('Unable to get Meeting bean.');
+        }
         $meeting->id = create_guid();
         $meeting->new_with_id = true;
         $event_local = $this->updateSuitecrmMeetingEvent($meeting, $event_remote);
```

Comment:
updateSuitecrmMeetingEvent() could return false as failure and this method returns it. Please update the phpdoc.

Reply:
Done